eos/constraints/B-to-psd-psd.yaml

```diff
@@ -0,0 +1,420 @@
+######################
+# B -> hh, h = pi, K #
+######################
+# {{{
+# B+ to pi0 pi+
+B^+->pi^+pi^0::BR_exp@PDG:2024A:
```

Comment:
What does the ``exp`` suffix to ``BR`` mean here in the constraint name and in the observable name below?

Reply:
That's the experimental BR, that accounts for the Bs mixing. For B+ and B0 decays, this is just the average BR (averaging B+ -> pi0 pi+, B+ -> pi0 pi+ and the charge conjugated modes).

Reply:
I'm happy to change the name or make it more explicit, though... This is also something we used in Bs -> phi ell ell, without naming it.